### DISCUSSION TITLE
Is it possible to use rooms in `python-socketio`?

### DISCUSSION THREAD

**Discussion by TechStudent10** [miguelgrinberg/python-socketio#779] [Q&A] [answered]
I am using a `Flask-SocketIO` server and I want to join a room on that server using `python-socketio`. I looked through the documentation but I didn't find anything. Is it even possible to join a room in `python-socketio`?

**Comment by miguelgrinberg** [comment, marked as answer]
Joining rooms is a server function, the client has to request this through an event to the server, then the server can put the client in a room. Since you are using Flask-SocketIO you should be able to use rooms just fine.

**Reply by TechStudent10** [reply]
Ohhhhhhhhhhhhh thanks.
